implementations/rust/ockam/ockam_api/src/authenticator/enrollment_tokens/acceptor.rs

```diff
@@ -46,14 +46,24 @@ impl EnrollmentTokenAcceptor {
 
         let token = match self.tokens.use_token(otc, now()?).await {
             Ok(Some(token)) => token,
-            Ok(None) | Err(_) => {
-                warn!("Unknown token received from {}", from);
+            Ok(None) => {
+                warn!("Unknown enrollment token received from {}", from);
                 return Ok(Either::Right(EnrollmentTokenAcceptorError(
-                    "Unknown token received from {}".to_string(),
+                    "Unknown enrollment token".to_string(),
+                )));
+            }
+            Err(err) => {
+                warn!(
+                    "Error using an enrollment token received from {}. Error: {}",
+                    from, err
+                );
+                return Ok(Either::Right(EnrollmentTokenAcceptorError(
+                    "Error using the enrollment token".to_string(),
```

Comment:
Why not surface the `err` here, so that we don't necessarily have to look for the logs?

Reply:
Returning unstructured server errors is a bad UX and potentially dangerous (can leak sensitive-ish data)